src/trie.rs

```diff
@@ -158,6 +197,44 @@ where
         }
     }
 
+    fn get_path_at<'a>(&self, n: &'a Node, partial: &Nibbles) -> TrieResult<Vec<Node>, C, D> {
```

Comment:
This lifetime `'a` is useless because there are no variables associated with it.